### ISSUE TITLE
Tool/Cmd to display chunks of a file

### ISSUE THREAD

**Issue by jmozah** [ethersphere/swarm#1501]
In many debugging situation, we need a command to display chunk level information of a given root hash. It would be good if other information like type of chunk (Tree chunk or Data chunk),  branch size , wether the chunk is local or not can be displayed along with the chunk hash.

````
swarm info <file hash> 

- abcdxxxx       16k   T   local                  
-- ayyyxxxx       16k    T   remote 
--- aabbddxx     4k      D  remote
--- bbbbddxx     4k      D  remote
--- ccbbddxx     4k      D  remote
--- ddbbddxx     4k      D  remote
          (up to 128 data chunks)
-- azzzxxxx       16k    T   remote 
--- aaaaaaaxx     4k      D  remote
--- bbbbbbxx     4k      D  remote 
--- cccccccxx     4k      D  remote
--- dddddddxx    4k      D  remote
        ( up to 128 data chunks)
````

Something like above  for the entire tree... 
If possible store time and access time would also be helpful.

**Comment by holisticode** [comment]
This functionality should only be available in a private network/cluster.

Endpoints should be responding optionally only; we can not count on a node in the main network telling you if it has a chunk or not. Privacy issues and surveillance/tracking resistance.

Having said that, we use similar functionality already via some kind of debug flag I believe. So only nodes which would have that flag on would respond (all nodes in a private test cluster for example)

**Comment by jmozah** [comment]
Good point. I definitely was not recommending a http api.. I was asking for
a rpc  API which is usually not open to internet by default..

There is a similar command which displays chunk info.. but that is for a
given file.. most of the gateways (like datafund or swarm) you don’t store
the original file so you don’t know the chunks it created..
